apps/web/src/utils/transformers.ts

```diff
@@ -0,0 +1,37 @@
+import type { Arrayified, Stringified } from "~/types";
+
+export function stringify<T>(v: T): Stringified<T> {
+  if (v instanceof Date || typeof v === "bigint") {
+    return v.toString() as Stringified<T>;
+  }
+
+  if (Array.isArray(v)) {
+    return v.map((item) => stringify(item)) as Stringified<T>;
+  }
+  if (v && typeof v === "object") {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const result: any = {};
+    for (const [key, value] of Object.entries(v)) {
+      result[key] = stringify(value);
+    }
+    return result;
+  }
```

Comment:
```suggestion
  if (v && typeof v === "object") {
    const result = {} as { [K in keyof T]: Stringified<T[K]> };
    for (const key in v) {
      const typedKey = key as keyof T;
      result[typedKey] = stringify(v[typedKey]);
    }
    return result as Stringified<T>;;
  }  
```
would be possible to avoid `any` type here?